test/test_ops.py

```diff
@@ -1155,6 +1156,68 @@ def _fn(*sample_args, **sample_kwargs):
                 graph = traced.graph_for(*inp)
                 FileCheck().check(op_name).check_not(variant_name).run(graph)
 
+class TestJitTEFuser(JitTestCase):
```

Comment:
I wonder if it would make sense to call this class something different so we can tell it apart from the one in `test_jit_fuser_te.py`?